pkgs/by-name/ba/basicswap/package.nix

```diff
@@ -0,0 +1,88 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+, linkFarm
+, secp256k1
+, particl-core
+, bitcoind
+, litecoind
+, namecoind
+, monero-cli
+, wownero
+  # , pivx
+}:
+
+let
+  secp256k1_anonswap = secp256k1.overrideAttrs (old: {
+    src = fetchFromGitHub {
+      owner = "tecnovert";
+      repo = "secp256k1";
+      rev = "anonswap_v0.2";
+      hash = "sha256-EeUyiYLmgWLQFm0ezLETJrOg3QeLHRx2YVbjwvPMB+Q=";
+    };
+  });
+  coincurve-anonswap = (python3Packages.coincurve.override {
+    secp256k1 = secp256k1_anonswap;
+  }).overrideAttrs (old: {
+    src = fetchFromGitHub {
+      owner = "tecnovert";
+      repo = "coincurve";
+      rev = "anonswap_v0.2";
+      hash = "sha256-EH8u8aZrBzM+TP2AN1zgtRgW8NMSVdfVQBUh6JQA3yE=";
+    };
+  });
+  bindir = linkFarm "bindir" (lib.mapAttrs (_: p: "${lib.getBin p}/bin") {
+    particl = particl-core;
+    bitcoin = bitcoind;
+    litecoin = litecoind;
+    namecoin = namecoind;
+    monero = monero-cli;
+    wownero = wownero;
+    # pivx = pivx;
```

Comment:
This should get a todo, otherwise it is a bit up to guessing what this was supposed to do.

Reply:
Added.